scripts/checkYaml.lean

```diff
@@ -0,0 +1,48 @@
+import Mathlib.Tactic.Basic
```

Comment:
Could you please add a copyright header and some docs to this file?

Reply:
Yes, but not until CI finished. It need to rebuild all of mathlib because I touched the lakefile...

Reply:
Done